mock-collector/src/main/java/org/apache/skywalking/plugin/test/mockcollector/mock/MockManagementService.java

```diff
@@ -23,12 +23,13 @@
 import org.apache.skywalking.apm.network.management.v3.InstancePingPkg;
 import org.apache.skywalking.apm.network.management.v3.InstanceProperties;
 import org.apache.skywalking.apm.network.management.v3.ManagementServiceGrpc;
+import org.apache.skywalking.plugin.test.mockcollector.entity.ValidateData;
 
 public class MockManagementService extends ManagementServiceGrpc.ManagementServiceImplBase {
 
     @Override
     public void reportInstanceProperties(InstanceProperties request, StreamObserver<Commands> responseObserver) {
-
+        ValidateData.INSTANCE.register();
```

Comment:
Do we verify this? Didn't see this in the expected file.

Reply:
We have some test cases of java plugin need to confirm plugin has registered. So mock collector provides this API.